var/spack/repos/builtin/packages/moab/package.py

```diff
@@ -58,7 +59,8 @@ class Moab(Package):
     depends_on('hdf5+mpi')
     depends_on('netcdf+mpi', when='+netcdf')
     depends_on('parmetis')
-    depends_on('trilinos')  # looking for zoltan.
+    depends_on('zoltan')
```

Comment:
```
depends_on('zoltan', when='+fortran')
```
for the reasons discussed in https://github.com/LLNL/spack/issues/2632#issuecomment-268043639